.golangci.yml

```diff
@@ -0,0 +1,24 @@
+run:
+  tests: false
+  timeout: 5m
+linters:
+  enable-all: true
+  disable:
+    - lll
+    - gocyclo
+    - interfacer
+    - unparam
+    - gocritic
+    - dupl
+    - errcheck
+    - gochecknoglobals
+    - funlen
+    - gocognit
+    - godox
+    - wsl
+    - exhaustivestruct
+    - nlreturn
+    - wrapcheck
+    - prealloc
+    - goerr113 # fixing this (returning wrapped static error types) should be a TODO at some point, not worrying about it now.
+    - cyclop
```

Comment:
I no longer use golangci-lint in my projects. I don't think it adds enough value to be worth it. Let's strip it out.